sample-action-client/src/extension.ts

```diff
@@ -20,6 +20,18 @@ export async function activate(context: vscode.ExtensionContext) {
         vscode.window.showInformationMessage(`Action ${action.name} was scheduled to run`,);
         basAPI.actions.performAction(action, {schedule: true});
     }));
+
+	context.subscriptions.push(vscode.commands.registerCommand("get.parameter", async () => {
+        const parameterName = await vscode.window.showInputBox({ prompt: "Enter Parameter Name", ignoreFocusOut: true });
+        if (!parameterName) { 
```

Comment:
it is better to avoid automatic boolean conversions in JavaScript
as it can lead to strange bugs in edge cases.

Something like:
```javascript
// undefined on escape and empty on enter
if (parameterName === undefined || parameterName === "") { /* ... */} 
```